backend/profiles/management/commands/createtestuser.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
```

Comment:
I think this is unnecessary; please pull in staging and delete this if that is the case.